src/components/DropdownSelect/ListBox.tsx

```diff
@@ -127,7 +127,9 @@ function Option({ item, state, inputValue }: OptionProps) {
     >
       <Flex $position={"relative"} $alignItems={"center"}>
         <OptionContext.Provider value={{ labelProps, descriptionProps }}>
-          {formatSchoolName(item.rendered, inputValue)}
+          {inputValue
+            ? formatSchoolName(item.rendered, inputValue)
+            : item.rendered}
```

Comment:
is there a test case we can add to capture the different use cases? i.e with and without input

Reply:
Yeah agreed, the naming of inputValue could be improved too. I say let's get the fix in and then write a ticket for that.